src/components/molecules/GameOptionCard/GameOptionCard.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import CardMedia from '@material-ui/core/CardMedia';
+import { makeStyles } from '@material-ui/core/styles';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardContent from '@material-ui/core/CardContent';
+import Typo from '../../atoms/Typo/Typo';
+
+const useStyles = makeStyles({
+  card: {
+    width: '350px',
+    height: '350px',
+    '&:hover': { color: '#208fea' },
+  },
+  cardActionMargin: {
+    marginTop: '1em',
+  },
+  image: {
+    width: '350px',
+    height: '200px',
+  },
+  marginBottom: {
+    marginBottom: '0.3em',
+  },
+});
+
+type GameOptionType = 'classic' | 'speed' | 'reverse' | 'watch';
+
+type GameOptionCardProps = {
+  option: GameOptionType,
+  onClick: React.MouseEventHandler<HTMLButtonElement>,
+};
+
+const GameOptionCard = ({ option, onClick } : GameOptionCardProps) => {
+  const classes = useStyles();
+
+  const ClassicGame = () => (
+    <>
+      <CardMedia
+        className={classes.image}
+        component="img"
+        alt="classic game"
+        image="/images/PongNormal.png"
+        title="classic game"
+      />
+      <CardContent>
+        <Typo className={classes.marginBottom} variant="h5">Classic Game</Typo>
+        <Typo variant="body1">상대방보다 먼저 5점을 획득하면 이기는 게임입니다. 키보드 방향키로 탁구채를 조작하여 공을 상대방쪽으로 넘기세요.</Typo>
+      </CardContent>
+    </>
+  );
```

Comment:
```suggestion
  const GameCard = ({ alt, src, title, description }) => ( // 이름은 임의로 지어봤어요
    <>
      <CardMedia
        className={classes.image}
        component="img"
        alt={alt}
        image={src}
        title={title}
      />
      <CardContent>
        <Typo className={classes.marginBottom} variant="h5">{title}</Typo>
        <Typo variant="body1">{description}</Typo>
      </CardContent>
    </>
  );
  ...
  const ClassicGame = () => <GameCard alt="..." src="..." title="..." description="..." />;
  const SpeedGame = () => <GameCard alt="..." src="..." title="..." description="..." />;
  const ReverseGame = () => <GameCard alt="..." src="..." title="..." description="..." />;
  const WatchGame = () => <GameCard alt="..." src="..." title="..." description="..." />;
```
이런 모양의 컴포넌트가 중복되고 있는데, GameOptionCard 바깥에 이런 식으로 재사용 가능한 컴포넌트를 만들어서 코드 중복을 줄여보는 건 어떨까요?

Reply:
오 이런 방법도 있군요 감사합니다:). 적용하고 푸쉬하겠습니다!!

Reply:
const GameCard = ({
    alt, src, title, description,
  }: GameCardType) => (
    <>
      <CardMedia
        className={classes.image}
        component="img"
        alt={alt}
        image={src}
        title={title}
      />
      <CardContent>
        <Typo className={classes.marginBottom} variant="h5">{title}</Typo>
        <Typo variant="body1">{description}</Typo>
      </CardContent>
    </>
  );

다음과 같이 GameCard를 만들어서 수정했습니다!